src/PKSim.Core/Mappers/IndividualToIndividualBuildingBlockMapper.cs

```diff
@@ -60,7 +64,17 @@ protected override IFormula TemplateFormulaFor(IParameter parameter, IFormulaCac
 
       protected override IReadOnlyList<IParameter> AllParametersFor(Individual individual)
       {
-         return individual.GetAllChildren<IParameter>().Where(x => x.GroupName != CoreConstants.Groups.RELATIVE_EXPRESSION).ToList();
+         bool shouldExportParameter(IParameter parameter)
+         {
+            //these parameters are exported separately
+            if (parameter.GroupName == RELATIVE_EXPRESSION)
+               return false;
+
+            return !_sameFormulaOrValueForAllSpeciesRepository.IsSameFormulaOrValue(parameter);
```

Comment:
my idea is that I am only going to export the parameter in the individual BB if it's not the same for all species (value or formula). Seems reasonable?

Reply:
yes